modules/migration-manually-rolling-back-migration.adoc

```diff
@@ -4,12 +4,75 @@
 // * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
 
 [id='migration-manually-rolling-back-migration_{context}']
-= Manually rolling back a migration
+= Rolling back a migration
 
 If your application was stopped during a failed migration with the {mtc-full} ({mtc-short}), you must roll it back manually in order to prevent data corruption in the persistent volume.
 
-This procedure is not required if the application was not stopped during migration, because the original application is still running on the source cluster.
+Rollback is not required if the application was not stopped during migration because the original application is still running on the source cluster.
 
+ifeval::["{mtc-version}" >= "1.4"]
+== Rolling back a migration in the web console
+
+You can roll back a migration in the web console.
+
+.Procedure
+
+. In the {mtc-short} web console, click *Migration plans*.
+. Click the Options menu {kebab} beside a migration plan and select *Rollback*.
```

Comment:
{kebab} --> :kebab:

Reply:
No, this is an attribute. not an image.

Reply:
It's listed as :kebab: in the common attribute file. And also noted in the guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc

Reply:
@sfortner-RH you use `:kebab:` to define the attribute (as is done in the common-attributes file), but when you actually use it in the text to be replaced, the syntax is `{kebab}`, as @apinnick has it here. 

Also, when reviewing, it is good to check the rendered version of the build. If you do that, then you should be able to see that the attribute is replaced correctly with the kebab icon.

Reply:
Thanks for clarifying @bergerhoffer